ts/components/ui/ListItemTransaction.tsx

```diff
@@ -64,24 +66,17 @@ type LeftComponentProps = {
   logoIcon: PaymentLogoIcon;
 };
 
-const IMAGE_LOGO_SIZE: IOIconSizeScale = 24;
-
-const isImageUri = (value: PaymentLogoIcon): value is ImageURISource =>
-  typeof value === "object" && value !== null && "uri" in value;
+const CARD_LOGO_SIZE: IOIconSizeScale = 24;
+const MUNICIPALITY_LOGO_SIZE = 44;
```

Comment:
Since this value is used as a width of the View that wraps the `LeftComponent`, what do you think to rename it to more generic `PAYMENT_LOGO_SIZE`?

Reply:
I believe naming it "MUNICIPALITY" serves more of a logic factor: since the municipality logo has no `size:number` prop, I am afraid an external reader would see it as an arbitrary number instead of a number used to standardize sizing; what do you think?